love/src/Config.js

```diff
@@ -436,6 +436,92 @@ export const severityEnum = {
   critical: 4,
 };
 
+// MT Camera Hexapod
+export const hexapodStatusStatetoStyle = {
+  UNKNOWN: 'undefined',
+  ENABLED: 'enabled',
+  DISABLED: 'disabled',
+  FAULT: 'fault',
+  OFFLINE: 'offline',
+  STANDBY: 'standby',
+};
+
+export const hexapodCommandableByDDSStateMap = {
+  false: 'NOT COMMANDABLE',
+  true: 'COMMANDABLE',
+};
+
+export const hexapodCommandableByDDSStatetoStyle = {
+  'NOT COMMANDABLE': 'alert',
+  COMMANDABLE: 'ok',
+};
+
+export const hexapodCompensationModeStateMap = {
+  false: 'DISABLED',
+  true: 'ENABLED',
+};
+
+export const hexapodCompensationModeStatetoStyle = {
+  DISABLED: 'undefined',
+  ENABLED: 'ok',
+};
+
+export const hexapodInterlockStateMap = {
+  false: 'DISENGAGED',
+  true: 'ENGAGED',
+};
+
+export const hexapodControllerStateMap = {
+  0: 'STANDBY',
+  1: 'DISABLED',
+  2: 'ENABLED',
+  3: 'OFFLINE',
+  4: 'FAULT',
+};
+
+export const hexapodControllerStatetoStyle = {
+  STANDBY: 'undefined',
+  DISABLED: 'undefined',
+  ENABLED: 'ok',
+  OFFLINE: 'undefined',
+  FAULT: 'warning',
```

Comment:
Are this `undefined` values correct? Why you are not using `warning`, `alert` or `critical` statuses?